packages/core/src/RenderingEngine/helpers/cpuFallback/rendering/generateLut.ts

```diff
@@ -36,9 +36,15 @@ export default function (
 
   const lut = image.cachedLut.lutArray;
 
-  const mlutfn = getModalityLut(image.slope, image.intercept, modalityLUT);
+  let mlutfn = getModalityLut(image.slope, image.intercept, modalityLUT);
   const vlutfn = getVOILUT(windowWidth, windowCenter, voiLUT);
 
+  if (image.isPreScaled) {
+    // if the image is already preScaled, it means the slop, intercept
```

Comment:
> it means the slop

typo

Reply:
Also shouldn't it be "the slope and the intercept", not "slope, intercept", there are only two items in the list.